tests/test_snapwell.py

```diff
@@ -1,16 +1,39 @@
 import os
 import unittest
 from datetime import datetime
+from os.path import abspath, join
+from unittest.mock import MagicMock
 
-from .testcase import TestCase
-from snapwell import SnapConfig
+import pytest
+from ecl.eclfile import EclFile
 
 # utils
-from snapwell import findRestartStep, roundAwayFromEven
-from snapwell import parse_date
+from snapwell import (
+    SnapConfig,
+    findKeyword,
+    findRestartStep,
+    parse_date,
+    roundAwayFromEven,
+    tryFloat,
+)
 
-from ecl.eclfile import EclFile
-from os.path import join, abspath
+from .testcase import TestCase
+
+
+def test_parse_not_a_date():
+    with pytest.raises(ValueError, match="datetime string on the form"):
+        parse_date("Not a date")
+
+
+def test_parse_not_a_float():
+    assert tryFloat("Not a float", 0.0) == 0.0
```

Comment:
I question the whole consept of `tryFloat`, as it is really a configuration issue, and should be dealt with much earlier, but you might deal with it in your other PR?

Reply:
I think different PR and definitely `tryFloat` is not the right concept here.